src/main/resources/assets/owo/lang/ru_ru.json

```
@@ -0,0 +1,55 @@
+{
+  "text.owo.itemGroup.tab_template": [
+    { "index": 0 },
+    { "text": " > ", "color": "gray" },
+    { "index": 1, "color": "dark_gray" }
+  ],
+  "text.owo.itemGroup.select_hint": {"text": "Шифт-кликните, чтобы выбрать несколько", "color": "gray"},
+  "text.owo.configure_hot_reload.title": "Настроить горячую перезагрузку",
```

Быть может, придираюсь, но буквальный перевод хотсваппинга как-то не идёт...
Могу предложить следующее:
```suggestion
  "text.owo.configure_hot_reload.title": "Настроить перезагрузку «на лету»",
```